samples/nrf9160/lwm2m_client/src/lwm2m/lwm2m_security.c

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2019 Foundries.io
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#define LOG_MODULE_NAME app_lwm2m_security
+#define LOG_LEVEL CONFIG_APP_LOG_LEVEL
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(LOG_MODULE_NAME);
+
+#include <zephyr.h>
+#include <net/lwm2m.h>
+
+#if defined(CONFIG_LWM2M_DTLS_SUPPORT)
+#include <net/tls_credentials.h>
+#include "nrf_inbuilt_key.h"
+
+#define TLS_TAG			16842753
```

Comment:
We should use a different tag here. This one is associated with a factory-assigned certificate/private key. I found the `asset_tracker` unable to connect after running lwm2m sample because it had PSK/identity assigned with the same tag - had to remove DTLS credentials manually.

Reply:
I reversed the tag to 35724861